aiohttp/connector.py

```diff
@@ -1113,6 +1123,9 @@ async def _wrap_create_connection(
                     interleave=self._interleave,
                     loop=self._loop,
                 )
+                if sock is not None:
```

Comment:
Coverage says that no tests hit the `is None` condition.
The return type for .start_connection() doesn't say it can return None, so I'm assuming this line should be removed?

Reply:
I'm guessing this line was added due to type errors caused by using the helper method. Now that the code happens inline, it shouldn't produce any type errors if removed.